test/fixtures/test-runner/output/hooks.js

```diff
@@ -166,5 +166,13 @@ test('t.after() is called if test body throws', (t) => {
   throw new Error('bye');
 });
 
+describe('run after when before throws', () => {
+  after(common.mustCall(() => {
+    console.log("- after() called")
+  }));
```

Comment:
You don't need the console log here, the must call is enough

Reply:
yes, but that shows unexpected behavior. :/
the tests pass for both common.mustCall() and common.mustNotCall() cb in after hook if we didn't pass any fn inside mustCall and mustNotCall. 
That's why I used console.log to test the actual expected behavior.

Reply:
Are you saying that the following code won't work?
```js
after(common.mustCall(() => {}));
```